deepchem/molnet/load_function/swissprot_datasets.py

```diff
@@ -0,0 +1,79 @@
+"""
+swissprot dataset loader.
+"""
+import os
+import deepchem as dc
+from deepchem.molnet.load_function.molnet_loader import TransformerGenerator, _MolnetLoader
+from deepchem.data import Dataset
+from typing import List, Optional, Tuple, Union
+
+SWISSP_URL = "https://deepchemdata.s3-us-west-1.amazonaws.com/datasets/uniprot_swissprot_14_06_21.csv"
+SWISSP_TASK=[]
+
+class _SWISSPROTLoader(_MolnetLoader):
+  
+  def create_dataset(self) -> Dataset:
+    dataset_file = os.path.join(self.data_dir, "uniprot_swissprot_14_06_21.csv")
+    if not os.path.exists(dataset_file):
+      dc.utils.data_utils.download_url(url=SWISSP_URL, dest_dir=self.data_dir)
+      dataset_file = os.path.join(self.data_dir, "uniprot_swissprot_14_06_21.csv")
+
+      
+    loader = dc.data.CSVLoader(
+        tasks=self.tasks, featurizer=self.featurizer, sanitize=True)
+    return loader.create_dataset(dataset_file, shard_size=8192)
+
+  codes = ['A', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'K', 'L',
+         'M', 'N', 'P', 'Q', 'R', 'S', 'T', 'V', 'W', 'Y']
+         
+  def load_swissprot(
+    featurizer: Union[dc.feat.Featurizer, str] = dc.feat.OneHotFeaturizer(codes),
+    splitter: Union[dc.splits.Splitter, str, None] = 'random',
+    #transformers: List[Union[TransformerGenerator, str]] = ['normalization'],
```

Comment:
I think you can uncomment this and just set the default value to `[]` to have no transformations

Reply:
Sure !